Gems/PhysX/Code/Tests/Benchmarks/PhysXBenchmarksUtilities.h

```diff
@@ -68,7 +68,8 @@ namespace PhysX::Benchmarks
             int benchmarkObjectType,
             GenerateColliderFuncPtr* genColliderFuncPtr = nullptr, GenerateSpawnPositionFuncPtr* genSpawnPosFuncPtr = nullptr,
             GenerateSpawnOrientationFuncPtr* genSpawnOriFuncPtr = nullptr, GenerateMassFuncPtr* genMassFuncPtr = nullptr,
-            GenerateEntityIdFuncPtr* genEntityIdFuncPtr = nullptr
+            GenerateEntityIdFuncPtr* genEntityIdFuncPtr = nullptr,
+            bool activateEntities = true
```

Comment:
nit: Not usually a fan of `bool` parameters 😝 but in this case I'll let it slide 😉

Reply:
Yeah, I think it's fine to keep it simple in this case.